public/app/features/connections/tabs/ConnectData/CardGrid/CardGrid.tsx

```diff
@@ -40,13 +49,24 @@ const getStyles = (theme: GrafanaTheme2) => ({
   }),
 });
 
-export type CardGridItem = {
-  id: string;
-  name: string;
-  description: string;
-  url: string;
+function PluginEnterpriseBadgeWithoutSignature() {
+  const customBadgeStyles = useStyles2(getBadgeColor);
```

Comment:
We added a `darkgrey` badge color last week, would that cover the need for custom badge styles here?

Reply:
I tried this but background color is different based on the theme.